ui/app/src/modules/Preview/PreviewConcierge.tsx

```diff
@@ -190,6 +192,15 @@ export function PreviewConcierge(props: any) {
       case 'craftercms.ice.assets':
         // TODO: aaron to fetch assets here...
         break;
+      case 'craftercms.ice.audiences':
+        if (
```

Comment:
Need to consider that there’s no errors. Otherwise it will loop the fetch.

Merge latest and see content types dispatch for example.